dag/archive/main.yml

```diff
@@ -188,6 +188,14 @@ steps:
   data://grapher/aviation_safety_network/2023-04-18/aviation_statistics:
     - data://garden/aviation_safety_network/2023-04-18/aviation_statistics
 
+  # World Bank GenderStats
+  data://meadow/wb/2023-12-29/gender_statistics:
+    - snapshot://wb/2024-12-29/gender_statistics.feather
+  data://garden/wb/2023-12-29/gender_statistics:
+    - data://meadow/wb/2023-12-29/gender_statistics
+  data://grapher/wb/2023-12-29/gender_statistics:
+    - data://garden/wb/2023-12-29/gender_statistics
+    -
```

Comment:
@veronikasamborska1994 there's an extra `-` that could be causing the error.

Reply:
yep, that was it... thank you!!